osu.Game/Overlays/FirstRunSetup/ScreenBehaviour.cs

```diff
@@ -0,0 +1,104 @@
+{
+    public class ScreenBehaviour : FirstRunSetupScreen
+    {
+        private SearchContainer<SettingsSection> searchContainer;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Content.Children = new Drawable[]
+            {
+                new OsuTextFlowContainer(cp => cp.Font = OsuFont.Default.With(size: 24))
+                {
+                    Text = FirstRunSetupOverlayStrings.BehaviourDescription,
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y
+                },
+                new GridContainer
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    ColumnDimensions = new[]
+                    {
+                        new Dimension(),
+                        new Dimension(GridSizeMode.Absolute, 10),
+                        new Dimension(),
+                    },
+                    RowDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
+                    Content = new[]
+                    {
+                        new[]
+                        {
+                            new TriangleButton
+                            {
+                                Anchor = Anchor.TopLeft,
+                                Origin = Anchor.TopLeft,
+                                Text = FirstRunSetupOverlayStrings.NewDefaults,
+                                RelativeSizeAxes = Axes.X,
+                                Action = applyStandard,
+                            },
+                            Empty(),
+                            new DangerousTriangleButton
+                            {
+                                Anchor = Anchor.TopRight,
+                                Origin = Anchor.TopRight,
+                                Text = FirstRunSetupOverlayStrings.ClassicDefaults,
+                                RelativeSizeAxes = Axes.X,
+                                Action = applyClassic
+                            }
+                        },
+                    },
+                },
+                searchContainer = new SearchContainer<SettingsSection>
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Children = new SettingsSection[]
+                    {
+                        new GeneralSection(),
+                        new SkinSection(),
+                        new UserInterfaceSection(),
+                        new GameplaySection(),
+                        new RulesetSection(),
+                        new AudioSection(),
+                        new GraphicsSection(),
+                        new OnlineSection(),
+                        new MaintenanceSection(),
+                        new DebugSection(),
+                    },
```

Comment:
While there may not be too much settings sections updates, I think it would be best to have this shared with the `CreateSections` method in `SettingsOverlay`.

Reply:
Intentionally left out input because I thought it was only the sub section, but turns out that's not the case. Probably best to share them all I guess (ignoring performance issues).

Reply:
Turns out sharing these is a bit of a pain, due to the `InputSection` sub-panel. Can definitely be resolved by restructuring the way the sub-panel is initialised, but probably for a follow-up effort?

I've added commenting for now so we don't accidentally get these out of sync for the time being. Not sure if it's enough to appease concerns.